test/backend/test_node.py

```diff
@@ -391,6 +391,25 @@ def test_global_average_pool(self):
         test_output[i1][i2][0][0] = sum / 6.
     np.testing.assert_almost_equal(output["Y"], test_output)
 
+  def test_image_sacler(self):
+    # Input:  (N x C x H x W), where N is the batch size,
+    # C is the number of channels, and H and W are the height
+    # and the width of the data
+    # Scale: (flout, default 1.0) the scale to apply
+    # Bias: applied to each channel, same size as C
+    # Output has same shape and type as input
+    x = self._get_rnd([1, 3, 224, 224])
+    #random distribution over [0,1), so add 0.1
+    scale = np.random.rand(1)[0] + 0.1
+    bias = np.random.rand(3)
+    node_def = helper.make_node("ImageScaler", ["X"], ["Y"], scale=scale, bias=bias)
+    output = run_node(node_def, [x])
+    test_out = np.multiply(x, scale)
+    test_out = np.transpose(test_out, [0,2,3,1])
```

Comment:
Maybe not formatted.
Could you fix this?

Reply:
OK